face/VatomView.js

```diff
@@ -0,0 +1,119 @@
+//
+//  BlockV AG. Copyright (c) 2018, all rights reserved.
+//
+//  Licensed under the BlockV SDK License (the "License"); you may not use this file or
+//  the BlockV SDK except in compliance with the License accompanying it. Unless
+//  required by applicable law or agreed to in writing, the BlockV SDK distributed under
+//  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+//  ANY KIND, either express or implied. See the License for the specific language
+//  governing permissions and limitations under the License.
+//
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable prefer-const */
+/* eslint-disable no-console */
+import FaceSelection from './FaceSelection';
+import ImageFace from './faces/ImageFace';
+
+
+// list registered faces
+let registeredFace = {
+  'native://image': ImageFace,
+};
+
+export default class VatomView {
+  constructor(bv, vAtom, FSP, config) {
+    this.blockv = bv;
+    this.vatomObj = vAtom;
+    this.fsp = FSP || FaceSelection.Icon;
+    this.config = config || {};
+    // eslint-disable-next-line
+    this._currentFace = {};
+
+    // create a default view with a div container
+    // eslint-disable-next-line
+    this.element = document.createElement('div');
+    this.element.style.position = 'relative';
+    this.element.style.width = this.config.width || '64px';
+    this.element.style.height = this.config.height || '64px';
+
+    // create loader
+    if (this.config.loader) {
+      this.loader = this.config.loader;
+    } else {
+      this.loader = document.createElement('div');
```

Comment:
The loader is not floating, it may interfere with the layout of other content inside the div... Maybe add this:

```
this.loader.style.cssText = 'position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; '
```